### PR TITLE
[zh-cn] update AggregateError

### PR DESCRIPTION
**Pull request by jasonren0403**
### Description

update main page and translated constructor page

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/aggregateerror/index.md [hunk 1]
```diff
@@ -3,42 +3,36 @@ title: AggregateError
...
-  - : 错误的描述，默认为空。
-- `message`{{Optional_Inline}}
-  - : AggregateError 错误的提示信息。
+## 实例属性
 
-## 描述
+_从父对象 {{jsxref("Error")}} 中继承实例属性。_
 
-一个`AggregateError`当需要由操作报告多个错误被抛出，例如通过[`Promise.any()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Promise/any)，在传递给它的所有[`Promise`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Promise/any)拒绝。
+以下属性在 `AggregateError.prototype` 上定义，并由所有 `AggregateError` 实例共享。
```
Comment (jasonren0403) [inline files/zh-cn/web/javascript/reference/global_objects/aggregateerror/index.md:21]:
```suggestion
以下属性在 `AggregateError.prototype` 上定义，并由所有 `AggregateError` 实例共享。
```

File: files/zh-cn/web/javascript/reference/global_objects/aggregateerror/aggregateerror/index.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+- `errors`
+  - : 一系列错误对象，实际上可能不是 {{JSxRef("Error")}} 的实例。
+- `message` {{optional_inline}}
+  - : 一个可选的对错误集合的可读描述。
+- `options` {{optional_inline}}
+  - : 一个具有以下属性的对象：
+    - `cause` {{optional_inline}}
+      - : 表示错误的具体原因的属性。
+        当捕捉并重新抛出一个带有更具体或有用的错误信息的错误时，这个属性可以用来传递原始错误。
```
Comment (fwqaaq) [inline files/zh-cn/web/javascript/reference/global_objects/aggregateerror/aggregateerror/index.md:34]:
```suggestion
      - : 表示错误的具体原因的属性。当捕捉并重新抛出一个带有更具体或有用的错误信息的错误时，这个属性可以用来传递原始错误。
```

File: files/zh-cn/web/javascript/reference/global_objects/aggregateerror/index.md [hunk 3]
```diff
@@ -3,42 +3,36 @@ title: AggregateError
...
+  - : 创建一个新的 `AggregateError` 对象
 
-- `errors`
-  - : 错误的描述，默认为空。
-- `message`{{Optional_Inline}}
-  - : AggregateError 错误的提示信息。
+## 实例属性
 
-## 描述
+_从父对象 {{jsxref("Error")}} 中继承实例属性。_
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/aggregateerror/index.md:19]: 父类

Comment (jasonren0403) [reply files/zh-cn/web/javascript/reference/global_objects/aggregateerror/index.md:19]:
```suggestion
_从父类 {{jsxref("Error")}} 中继承实例属性。_
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**